generateWebPage.py

```diff
@@ -174,7 +174,7 @@ def createYearButton(spex, year):
     yearTITLE = re.sub(r"[-]*.txt", "", yearTITLE)
 
     titleBox = '''<div class="sb"
-    style="background-color:#1a1a1a;color:white;text-align:left;font-weight:bold;font-size:20;border-left:2px-solid-red;padding:5px;">{}</div>
+    style="background-color:#1a1a1a;color:white;text-align:left;font-weight:bold;font-size:20;border-left:2px-solid-red;padding:5px;margin-left:0px;">{}</div>
```

Comment:
This class gets duplicated at least 66 times in the generated html and contains 2 errors.
I'd like to see it moved out from the script and into the template instead of further extending it in the script.

I'd suggest either to fix it with this PR or create an issue for it in the main repo.

Reply:
To save a byte or two you could also exclude the px and write margin-left:0